lib/hanami/cli/generators/app/slice.rb

```diff
@@ -14,23 +14,24 @@ def initialize(fs:, inflector:)
             @inflector = inflector
           end
 
-          def call(app, slice, slice_url_prefix, context: SliceContext.new(inflector, app, slice, slice_url_prefix)) # rubocop:disable Metrics/AbcSize
-            fs.inject_line_before_last(fs.join("config", "routes.rb"), /end/, t("routes.erb", context).chomp)
+          def call(app, slice, slice_url_prefix, context: SliceContext.new(inflector, app, slice, slice_url_prefix))
+            fs.inject_line_at_block_bottom(fs.join("config", "routes.rb"), /define/, t("routes.erb", context).chomp)
```

Comment:
Since the `slice_url_prefix` is optional, should we make this conditional on `slice_url_prefix` being truthy?

UPDATE: I've looked through the code a bit more and I see in `Commands::App::Generate::Slice` that we actually default the slice prefix to `"/"` if it isn't otherwise specified. I wonder if it is actually helpful to let the nil go all the way through so we can skip adding routes sections if slices are intended to be "headless"? What do you think?

Reply:
@timriley If you don't specify the option, it generates a prefix that is the underscored version of the slice name.